astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -410,3 +410,30 @@ def test_icrs_altaz_moonish(testframe):
 
     #now check that the distance change is similar to earth radius
     assert 1000*u.km < np.abs(moonaa.distance - MOONDIST).to(u.au) < 7000*u.km
+
+
+@remote_data
+def test_gcrs_self_transform_closeby():
+    """
+    Tests GCRS self transform for objects which are nearby and thus
+    have reasonable parallax.
+
+    Create moon positions using JPL DE432s ephemeris (small download).
+
+    The two lunar positions (one geocentric, one at a defined location)
+    are created via a transformation from ICRS to two different GCRS frames.
+
+    We test that the GCRS-GCRS self transform can correctly map one GCRS
+    frame onto the other.
+    """
+    t = Time("2014-12-25T07:00")
+    moon_geocentric = get_moon(t, ephemeris='de432s')
```

Comment:
Oh and in case it's not clear, not having `remote_data` is good b/c travis doesn't run remote data tests, among other things.

Or if you want, @StuartLittlefair, you might also just add a second test along the lines I said above and leave this in.